pymatgen/command_line/chargemol_caller.py

```diff
@@ -179,14 +177,13 @@ def _execute_chargemol(self, **jobcontrol_kwargs):
             jobcontrol_kwargs: Keyword arguments for _write_jobscript_for_chargemol.
         """
         with ScratchDir("."):
-            with zopen(self._chgcarpath, "rt") as f_in, open("CHGCAR", "w") as f_out:
-                shutil.copyfileobj(f_in, f_out)
-            with zopen(self._potcarpath, "rt") as f_in, open("POTCAR", "w") as f_out:
-                shutil.copyfileobj(f_in, f_out)
-            with zopen(self._aeccar0path, "rt") as f_in, open("AECCAR0", "w") as f_out:
-                shutil.copyfileobj(f_in, f_out)
-            with zopen(self._aeccar2path, "rt") as f_in, open("AECCAR2", "w") as f_out:
-                shutil.copyfileobj(f_in, f_out)
+            try:
+                os.symlink(self._chgcarpath, "./CHGCAR")
+                os.symlink(self._potcarpath, "./POTCAR")
+                os.symlink(self._aeccar0path, "./AECCAR0")
+                os.symlink(self._aeccar2path, "./AECCAR2")
+            except OSError as e:
+                print(f"Error creating symbolic link: {e}")
```

Comment:
Using `os.symlink` here seems strange. Why not read the files from where they are?

Reply:
`chargemol` is different. It seems like here is no input arguments to indicate where the input files (e.g. CHGCAR AECCAR0) stored, so we need to all the input file in ScratchDir. To avoid copying, just use symbolic link can "pretend" there are in the same directory but actually they are just the shortcut linked to the file in the original place.